fdmi/plugins/fdmi_plugin_st.sh

```diff
@@ -58,7 +58,7 @@ do_some_kv_operations()
 			    -f $M0T1FS_PROC_ID -s "
 
 		echo "Let's create an index and put {somekey:somevalue}"
-		"$M0_SRC_DIR/utils/m0kv" "${MOTR_PARAM}"                                       \
+		"$M0_SRC_DIR/utils/m0kv" ${MOTR_PARAM}                                       \
```

Comment:
Please note, this Codacy warning is reviewed and is false.
${MOTR_PARAM} will be passed to the command 'm0kv' as multiple arguments.
If it is quoted, it will be treated as a single argument.